python/experimental/tests/eager/test_kernel_features_eager.py

```diff
@@ -104,7 +104,11 @@ def test_grover():
 
     @cudaq.kernel
     def reflect(qubits):
-        ctrls = qubits.front(qubits.size() - 1)
+        #[RFC]: Using index with 'front'. such as 'qubits.front(qubits.size() - 1)' 
```

Comment:
This issue was introduced with the update to pybind 2.11 (appears to be a pybind bug). pybind version 2.10 and below work. Easiest fix would be to lower to pybind 2.10 until a fix appears in a future version.

Reply:
This error shows up with pybind11 v2.11.0+. `qview` bindings work with pybind11 v.10.4 and earlier.
Looking at changelog, pybind11 v2.11.0 introduced https://github.com/pybind/pybind11/pull/4631 and https://github.com/pybind/pybind11/pull/4623. These changes recursively check if a type is movable. 
Previously pybind11 relied on `std::is_move_constructible`, now, it has own recursive implementation.
Refer test case: https://github.com/pybind/pybind11/blob/master/tests/test_copy_move.cpp#L383
`qview` is copyable but, `qvector` and `qudits` are non-copyable and non-movable. Also, not sure if related, but, std::span isn't supported in default bindings. (https://github.com/pybind/pybind11/discussions/4177).

Reply:
Lowered pybind11 to v2.10.4 and captured behavior in [issue#1046](https://github.com/NVIDIA/cuda-quantum/issues/1046).